v4-client-js/src/clients/modules/account.ts

```diff
@@ -121,11 +160,40 @@ export default class AccountClient extends RestClient {
     });
   }
 
+  async getParentSubaccountOrders(
```

Comment:
seems like the naming scheme here is getParentSubaccountNumber{XYZ}, can we match that?

Reply:
yep